src/main/java/com/dumbdogdiner/stickyapi/bungeecord/command/builder/CommandBuilder.java

```diff
@@ -0,0 +1,226 @@
+/**
+ * Copyright (c) 2020 DumbDogDiner <dumbdogdiner.com>. All rights reserved.
+ * Licensed under the MIT license, see LICENSE for more information...
+ */
+package com.dumbdogdiner.stickyapi.bungeecord.command.builder;
+
+import com.dumbdogdiner.stickyapi.StickyAPI;
+import com.dumbdogdiner.stickyapi.bungeecord.util.SoundUtil;
+import com.dumbdogdiner.stickyapi.common.arguments.Arguments;
+import com.dumbdogdiner.stickyapi.common.command.ExitCode;
+import com.dumbdogdiner.stickyapi.common.command.builder.CommandBuilderBase;
+import com.dumbdogdiner.stickyapi.common.util.NotificationType;
+import com.dumbdogdiner.stickyapi.common.util.StringUtil;
+import com.google.common.collect.ImmutableList;
+import net.md_5.bungee.api.CommandSender;
+import net.md_5.bungee.api.ProxyServer;
+import net.md_5.bungee.api.connection.ProxiedPlayer;
+import net.md_5.bungee.api.plugin.Command;
+import net.md_5.bungee.api.plugin.Plugin;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.*;
+import java.util.concurrent.FutureTask;
+
+// TODO make a bungee command class that this works with
+public class CommandBuilder extends CommandBuilderBase<CommandBuilder> {
```

Comment:
This has been completely overhauled in #61, recommend not merging this + related files.

Reply:
Please see changes in #61 and make changes based on that, I want to ensure there are no major conflicts <3

Reply:
I'll be combining and doing another rewrite most likely